DigitalLearningSolutions.Data/Models/SectionContent/SectionContent.cs

```diff
@@ -2,20 +2,26 @@
 {
     public class SectionContent
     {
+        public string CustomisationName { get; }
+        public string ApplicationName { get; }
```

Comment:
I think this should be joined together as Title, like it is for course content 🙂